docs/release_notes/0.6.0.md

```diff
@@ -0,0 +1,7 @@
+# Release 0.6.0
+
+## Features
+
+- Rename experimental command `eksctl gitops apply` to `eksctl enable profile` (#1347) and use a temporary directory when cloning the repository (#1337)
```

Comment:
We've been using the convention of not capitalising the first word:

```suggestion
- rename experimental command `eksctl gitops apply` to `eksctl enable profile` (#1347) and use a temporary directory when cloning the repository (#1337)
```

Also, any reason why we can't have #1337  as its own item, perhaps under "Code Improvements"?